action-server/covidflow/utils/persistence.py

```diff
@@ -79,6 +79,48 @@ def save_assessment(slot_values: Dict[Text, Any], reminder_id=None):
         session.close()
 
 
+def get_reminder(slot_values: Dict[Text, Any]):
+    logger.debug("Fetching reminder")
+
+    session = session_factory()
+    try:
+        reminder_id = _get_reminder_id(slot_values)
+        reminder = session.query(Reminder).get(reminder_id)
+        if reminder is None:
+            raise Exception("Reminder does not exist")
+
+    except:
+        raise
```

Comment:
This seems odd...

Reply:
Yes but we have nothing special to do with the error, just re-throwing it (with this wonderfully simple syntax), the try-except pair is there for the finally clause only, to make sure the session closes.